### PR TITLE
Added error handling to CLI submit

### PR DESCRIPTION
**Pull request by ojalatodd**
Added needed error handling to CLI
submit command.

Signed-off-by: Todd Ojala <todd@bitwise.io>

### CODE REVIEW

File: cli/sawtooth_cli/submit.py [hunk 1]
```diff
@@ -34,16 +35,23 @@ def _split_batch_list(args, batch_list):
...
+        raise CliException(e)
 
     rest_client = RestClient(args.url)
 
     start = time.time()
 
     for batch_list in _split_batch_list(args, batches):
-        rest_client.send_batches(batch_list)
+        try:
+            rest_client.send_batches(batch_list)
```
Comment (peterschwarz) [inline cli/sawtooth_cli/submit.py]: The rest client already throws CliExceptions for the correct events.

File: cli/sawtooth_cli/submit.py [hunk 2]
```diff
@@ -34,16 +35,23 @@ def _split_batch_list(args, batch_list):
...
 def do_submit(args):
-    with open(args.filename, mode='rb') as fd:
-        batches = batch_pb2.BatchList()
-        batches.ParseFromString(fd.read())
+
+    try:
+        with open(args.filename, mode='rb') as fd:
+            batches = batch_pb2.BatchList()
+            batches.ParseFromString(fd.read())
+    except FileNotFoundError as e:
```
Comment (peterschwarz) [inline cli/sawtooth_cli/submit.py]: This should can probably be generalized to `IOError`

File: cli/sawtooth_cli/submit.py (hunk 1 above)
Comment (ojalatodd) [inline cli/sawtooth_cli/submit.py]: Right. Testing, though, it worked better to have the exception passed up. Will chat on slack.

Comment (ojalatodd) [inline cli/sawtooth_cli/submit.py]: Fixed

File: cli/sawtooth_cli/submit.py (hunk 2 above)
Comment (ojalatodd) [inline cli/sawtooth_cli/submit.py]: Changed.

**Pull request merged**